.github/workflows/scripts/slack_gem_notifications.rb

```diff
@@ -0,0 +1,42 @@
+# encoding: utf-8
+# This file is distributed under New Relic's license terms.
+# See https://github.com/newrelic/newrelic-ruby-agent/blob/main/LICENSE for complete details.
+
+require 'httparty'
+
+# TODO: Add support for comparing latest and previous ruby versions
+# TODO: If possible, specify how many gem objects we recieve from Ruby Gems API. We don't
+# need a gem's entire history.
+
+WATCHED_GEMS = ARGV
+
+if WATCHED_GEMS.empty?
+  warn 'WATCHED_GEMS cannot be empty'
+  exit
+end
+
+def watch_gem(gem_name)
+  gem_info = HTTParty.get("https://rubygems.org/api/v1/versions/#{gem_name}.json")
+  latest_versions = []
+  gem_info.each do |gem|
+    latest_versions << gem if gem['platform'] == 'ruby'
+    break if latest_versions.size == 2
```

Comment:
Why is this break statement here? What happens if `latest_versions.size == 2` that would make us want to exit?

Reply:
^-- This is a great question for which it would be good to have a comment based answer to.

Reply:
It is an answer I will comment! But also, the [Ruby Gems API](https://guides.rubygems.org/rubygems-org-api/) returns a large array of hashes (which each represents a release). Array entries are in order by the date they were released, the latest appearing first.

The break limits us to the only 2 ruby-platform items/hashes we want: the newest version and the previous version.

There's a TODO to revisit limiting the API response to how much we're getting back, but right now that isn't supported.